package.json

```diff
@@ -42,7 +42,7 @@
     "codeclimate-test-reporter": "^0.5.0",
     "ember-cli": "^3.10.1",
     "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-addon-docs": "^0.6.13",
+    "ember-cli-addon-docs": "^0.6.*",
```

Comment:
I think this change is not needed as the `^` in semver allows the 3rd value to drift anyways.

Is is possible the important part of this PR is actually only that the lock file was regenerated?

Reply:
Thank you for pointing out. Made the pr just to update yarn.lock